src/assets/styles/app.css

```diff
@@ -0,0 +1,265 @@
+:root {
+  --menu-bg-width: 50rem;
+  --menu-bg-width-phone: 320px;
+  scroll-behavior: smooth;
+}
+
+@font-face {
+  font-family: 'BD Northern';
+  src: url('fonts/ByDesignNorthern.ttf') format('truetype');
+  font-weight: normal;
+  font-style: normal;
+  font-display: swap;
+}
+
+body {
+  margin: 0;
+  background-color: #151311;
+  color: rgb(148, 133, 0);
+  line-height: 1.5;
+  font-family: 'BD Northern', sans-serif;
+}
+
+h1 {
+  color: rgb(207, 197, 152);
+}
+
+h2 {
+  color: rgb(202, 194, 84);
+  size: 30px;
+}
+
+h5 {
+  color: rgb(215, 211, 171);
+}
+
+footer {
+  color:#ab9852;
+  text-align: center;
+  font-size: 20px;
+  display: flex;
+  flex-direction: column;
+}
+
+.header {
+  width: 100%;
+  height: 594px;
+  position: relative;
+  background-image: url(https://grizzly.s.cz/img/Bear_Skull.jpg);
```

Comment:
You should ideally point to the images in your assets directory (like `'../images/Bear_Skull.jpg'`), not to an absolute URL.